libvast/vast/concept/parseable/vast/endpoint.hpp

```diff
@@ -32,11 +33,7 @@ struct endpoint_parser : parser<endpoint_parser> {
     using namespace parser_literals;
     auto hostname = +(alnum | chr{'-'} | chr{'_'} | chr{'.'});
     auto host = hostname ->* [&](std::string x) { e.host = std::move(x); };
-    auto port = (':' >> u16) ->* [&](uint16_t x) { e.port = x; };
-    auto p
-      = (host >> ~port)
-      | port
-      ;
+    auto p = (host >> ~(':' >> parsers::port)) | ':' >> parsers::port;
```

Comment:
A port parser has the form `x/T` where `x` is the number and `T` the port type. This is our ASCII data representation. Is that intentional? I could imagine, because you may want to be able to define UDP, but I'm not sure if this changes the syntax on the command line. In any case, we should probably update the unit tests (`endpoint.cpp`) as well if the suffix is now mandatory.

If the syntax for passing an endpoint changes, this is also a user-facing change needs documentation in the changelog.

Reply:
I should have run the tests before opening this PR...
The parser should still be able to accept the original syntax. This is a mistake on my part and i am looking into possible solutions atm.